### PR TITLE
[tests-only][full-ci] add test to check activity after language translation

### PR DESCRIPTION
**Pull request by prashant-gurung899**
## Description
This PR adds test to check activity after changing the systems language to your preferred one.(German 'de' in this case).
``` feature
Scenario: check activity message with different language
```

## Related Issue

- Related to : https://github.com/owncloud/ocis/issues/9543
- Bug Report : https://github.com/owncloud/ocis/issues/9856

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiActivities/shareActivities.feature [hunk 1]
```diff
@@ -1035,3 +1035,97 @@ Feature: check share activity
         }
       }
       """
+
+  @issue-9856
+  Scenario: check activitiy message after language translation
```
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/shareActivities.feature:1040]: Also, please move the scenario to another activities file.

Comment (prashant-gurung899) [reply tests/acceptance/features/apiActivities/shareActivities.feature:1040]: @saw-jan Sure, any suggestions for the another activities file's name?

Comment (saw-jan) [reply tests/acceptance/features/apiActivities/shareActivities.feature:1040]: May be just move `apiGraph/activities.feature` to `apiActivities/activities.feature` and add the scenarios there

**Review by saw-jan — APPROVED** [review]
:+1:

**Pull request merged**